sk0-front/src/app/global-services/notification.service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { ToastController } from '@ionic/angular';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class NotificationService {
+  constructor(private toastController: ToastController) {}
+
+  async toastMessage(message: string, position?: 'top' | 'middle' | 'bottom') {
```

Comment:
Good question!
If I remember right, in the toastController.create method if the position is undefined it will be 'bottom' by default, thats why I made the position parameter of the toastMessage optional.

Reply:
I checked it, it really puts it at the bottom by default, so that's good this way :)